crates/holochain_p2p/Cargo.toml

```diff
@@ -37,6 +37,8 @@ tokio-stream = "0.1"
 kitsune2_api = "0.0.1-alpha.1"
 bytes = "1.10"
 holochain_sqlite = { version = "0.5.0-dev.16", path = "../holochain_sqlite" }
+lair_keystore_api = "=0.5.3"
```

Comment:
Does this have to be pinned?

Reply:
short: yep : )
long: we don't have comprehensive enough tests in lair that verify functionality with previous lair versions, so the client won't connect to a server if even the patch differs.

Reply:
Not necessarily but it's pinned everywhere else so I'm just following the pattern. It's pinned in `kitsune_p2p` and in `lair_keystore`. So even if we didn't pin here too, it's pinned. I'm assuming it's better error messaging to be consistent.